profiles/killbill/src/test/java/org/killbill/billing/server/log/obfuscators/TestPatternObfuscator.java

```diff
@@ -17,16 +17,112 @@
 
 package org.killbill.billing.server.log.obfuscators;
 
-import org.killbill.billing.server.log.ServerTestSuiteNoDB;
+import java.util.Collection;
+import java.util.List;
+
 import org.mockito.Mockito;
 import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 import ch.qos.logback.classic.spi.ILoggingEvent;
 
-public class TestPatternObfuscator extends ServerTestSuiteNoDB {
+public class TestPatternObfuscator {
```

Comment:
Could you add the following comments so we don't forget and don't try to 'fix it':

1. On top of the class: `// Test does not extend base class on purpose so as to not pull the static Logger that would be initialized too early`
2.  `public class TestPatternObfuscator { /* extends ServerTestSuiteNoDB */`

Reply:
Done